gunicorn/workers/gthread.py

```diff
@@ -156,18 +156,17 @@ def murder_keepalived(self):
                 with self._lock:
                     self._keep.appendleft(conn)
                 break
-            else:
-                self.nr_conns -= 1
-                # remove the socket from the poller
-                with self._lock:
-                    try:
-                        self.poller.unregister(conn.sock)
-                    except EnvironmentError as e:
-                        if e.errno != errno.EBADF:
-                            raise
-                    except KeyError:
-                        # already removed by the system, continue
-                        pass
+            self.nr_conns -= 1
```

Comment:
An empty line might make this more readable to me.

Reply:
Done.